build-config/make/mtdutils.make

```diff
@@ -24,7 +24,7 @@ MTDUTILS_STAMP		= $(MTDUTILS_SOURCE_STAMP) \
 			  $(MTDUTILS_BUILD_STAMP) \
 			  $(MTDUTILS_INSTALL_STAMP)
 
-MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo
+MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo ubiattach ubimkvol ubidetach ubirmvol
```

Comment:
Hmmm.  So the proposal here is if one wants the full implementation of the tools they should remove the config from busybox locally.  A little goofy.

However, a lot of the old powerpc machines are really resource constrained, so it is best to not increase their image size.

I'm OK with this proposal.